### ISSUE TITLE
Error after installing FBP and entering the twilight forest dimension

### ISSUE THREAD

**Issue by THHakureiReimu** [TominoCZ/FancyBlockParticles#103]
## Steps to reproduce issue:
Install FBP-2.4.0 and Twilight Forest 2.3.7,an error occurred while entering the twilight forest dimension.
No crash, just show "A fatal error has occurred this connection is terminated".

## MC Version:
Minecraft 1.7.10

## FBP Version:
2.4.0

## Forge Version (e.g. 2604):
10.13.4.1614

**Comment by THHakureiReimu** [comment]
Install only FBP-2.4.0 and Twilight Forest 2.3.7,error as usual.

**Comment by TominoCZ** [comment]
Actually, if you look into the output log, you'll see this: https://pastebin.com/FCHFEzYL
This tells me that it's not a bug in my mod, but rather in TwilightForest.

**Comment by Pokihead** [comment]
I have this problem too :( but nice to know how to get it work temporarily.
Does Fancy Rain these bigger quadratic rain blocks?

**Comment by TominoCZ** [comment]
What do you mean?

**Comment by Pokihead** [comment]
I mean that I had the same problem as THHakureiReimu, that MC crashes into main menu if i travel in the twilight forest with FancyBlockParticles 2.4.1
After a lot of testing I discovered that it is the combination of your mod an Twilight Forest which is causing the problem. Then I found this thread here and your suggestion to deactivate Fancy Rain and Snow. This is working for me and I only wanted to know if fancy rain does that the rain looks more like MC water blocks so like big drops of rain :D

Should I report this Error to the devs of  Twilight Forest too? But i dont think they will fix it because the last update for 1.7.10 was 2015 ^^

**Comment by TominoCZ** [comment]
Yeah.. the rain is little cubes falling from the sky... The cubes have a color set depending on the sky color....you can just see it for yourself in the normal world, can't ya? XD

Definitely report this issue to the devs of TwilightForest, I can't do much to fix it.

**Comment by Pokihead** [comment]
yes sure but i had to disable them because of Twilight Forest so i dont see them anymore xD
ok i will report it there. Can I relink youre pastebin? Because I never got a crash log because it crashed only to main menu
